source/extensions/filters/http/ext_authz/ext_authz.cc

```diff
@@ -346,6 +346,8 @@ void Filter::onComplete(Filters::Common::ExtAuthz::ResponsePtr&& response) {
       config_->httpContext().codeStats().chargeResponseStat(info, false);
     }
 
+    decoder_callbacks_->streamInfo().setResponseFlag(
```

Comment:
This makes sense. 

Optional: Mind to add a comment for future readers, that is why we set it prior to calling `sendLocalReply`.

Reply:
also can put setResponseFlag and sendLocalReply into a inline method, but currently this is enough for me.